tests/common.rs

```diff
@@ -16,14 +16,19 @@ macro_rules! tests {
 
         #[test]
         fn wrong_address_raises_error() {
+            let mut below_min_mock = I2cMock::new(&[]);
```

Comment:
Could you split this test function into two?

Reply:
Done!